cl/citations/management/commands/westlaw_citation_merger.py

```diff
@@ -0,0 +1,354 @@
+            # citation and parallel citation
+            results_names = citation_cluster_results.values_list(
+                "case_name", flat=True
+            )
+
+            results_ids = citation_cluster_results.values_list("pk", flat=True)
+
+            parallel_results_names = (
+                parallel_citation_cluster_results.values_list(
+                    "case_name", flat=True
+                )
+            )
+
+            parallel_results_ids = (
+                parallel_citation_cluster_results.values_list("pk", flat=True)
+            )
+
+            if results_count == 1 and parallel_results_count == 0:
+                # Add parallel citation to cluster
+                add_citation(
+                    parallel_citation,
+                    citation_cluster_results.first().pk,
+                    debug,
+                )
+            elif results_count == 0 and parallel_results_count == 1:
+                # Add citation to cluster with parallel citation.
+                # Note: there may be the case that we can have duplicate
+                # cases, but one has date_filed_is_approximate and the
+                # other not, we will add the citation to the case with
+                # date_filed_is_approximate is false, e.g. cluster id:
+                # 7414160 and 5350679
+                add_citation(
+                    citation,
+                    parallel_citation_cluster_results.first().pk,
+                    debug,
+                )
+            elif results_count == 1 and parallel_results_count == 1:
+                # The case could be duplicated, compare the ids of each
+                # result
+                if (
+                    citation_cluster_results.first().pk
+                    != parallel_citation_cluster_results.first().pk
+                ):
+                    logger.warning(
+                        f'Possible duplicated cases for citations: "{citation}" with clusters: {citation_cluster_results.first().pk} and {parallel_citation_cluster_results.first().pk}'
+                    )
+                else:
+                    logger.warning(
+                        f'Cluster {citation_cluster_results.first().pk} already have both citations "{citation}" and "{parallel_citation}"'
+                    )
+            elif results_count > 1 and parallel_results_count == 0:
+                # We got more than one result using the citation, we can
+                # try to find the case name using csv data and try to
+                # match it with the correct case
+
+                # Check if case name from csv is exactly the same in
+                # case_names list and get index, this is just a workaround
+                # to use as much of westlaw data as possible, we could try
+                # to implement something more complex
+                indexes_found = find_indexes(list(results_names), case_name)
```

Comment:
I think we should be doing this anyway.  there are too many duplicate case citations to not do a little checking.